machina/templates/machina/partials/form_field.html

```diff
@@ -2,10 +2,16 @@
 {% load widget_tweaks %}
 
 <div id="div_id_{{ field.html_name }}" class="form-group{% if field.errors %} has-error{% endif %}">
+  {% if not field|widget_type == 'checkboxinput' %}
```

Comment:
I can understand the reasons behind this but this change introduces useless logic just to use a single "partial template" to render form fields. Why not just create a new "form_field_checkbox.html" partial that you could include when you need to display checkboxes? You can also render your checkboxes directly in the form without using a partial (this is done in other templates in this project).